posts/oversubscribed-tracks.md

```diff
@@ -0,0 +1,13 @@
+One of the biggest challenges we face is when tracks get too popular for our mentors to keep up. The wait time for students increases, and mentors have the very unsatisfying experience of seeing queues keep growing despite putting a huge amount of work in to try and keep on top.
+
+Solving this is far from trivial, but we believe we have some good solutions that will make things better (see our [recent blog post](https://exercism.io/blog/whats-next-for-exercism-aug-2019) for details). However, these will all take time to implement, and there is a very real problem today.
+
+We've therefore decided to add an "Oversubscribed" mode to tracks, where when the median wait time for core exercises becomes too long, we:
+- Restrict all new students joining a track to use Practice Mode
+- Disallow any new submissions in Practice Mode
+
+Once a track gets it's median wait time back under control, the system will automatically allow new students to join Mentored Mode, and allow Practice Mode students to submit.
```

Comment:
```suggestion
Once a track gets its median wait time back under control, the system will automatically allow new students to join Mentored Mode, and allow Practice Mode students to submit.
```

Reply:
One extra question here: above you refer to disallowing `new submissions`, here you refer to allowing `new students` again. Would a student that's already signed up to Mentored Mode be allowed to continue submitting?

Reply:
Clarified below about existing students.

No new submissions in Practice mode. No new students in Mentored Mode.